chains/xcm-integration-tests/emulated/chains/relays/rococo/Cargo.toml

```diff
@@ -0,0 +1,52 @@
+[package]
+name = "rococo-emulated-chain"
+description = "Rococo emulated chain"
+edition = "2021"
+license = "GPL-3.0-only"
+publish = false
+version = "0.0.0"
+[package.authors]
+workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+# Substrate
+cumulus-primitives-core = { workspace = true }
+polkadot-parachain-primitives = { workspace = true }
+sc-consensus-grandpa = { workspace = true }
+sp-consensus-babe = { workspace = true }
+sp-consensus-beefy = { workspace = true, default-features = true }
+sp-core = { workspace = true }
+
+# Polkadot
+rococo-runtime = { workspace = true }
+rococo-runtime-constants = { workspace = true }
+
+# Cumulus
+emulated-integration-tests-common = { workspace = true }
+runtime-parachains = { workspace = true }
+tanssi-emulated-integration-tests-common = { workspace = true }
+xcm-emulator = { workspace = true }
+
+
+[features]
+default = [ "std" ]
```

Comment:
I dont see a reason why to put all these packages under std features, as they are only used in tests. See https://github.com/paritytech/polkadot-sdk/blob/master/cumulus/parachains/integration-tests/emulated/chains/relays/rococo/Cargo.toml

Reply:
Removed all the std-s. We need std only in several places for the runtimes.